aws/utils.go

```diff
@@ -145,4 +145,8 @@ func getLastPathElement(path string) string {
 
 func lastPathElement(_ context.Context, d *transform.TransformData) (interface{}, error) {
 	return getLastPathElement(types.SafeString(d.Value)), nil
-}
\ No newline at end of file
+}
+
+func convertSafeString(_ context.Context, d *transform.TransformData) (interface{}, error) {
+	return types.SafeString(d.Value), nil
+}
```

Comment:
Why we need this `*string` to `string` conversion here?
You can use `transform.EnsureStringArray` function directly

Reply:
arnToakas used as discuss